keyboards/chouchou/README.md

```diff
@@ -0,0 +1,41 @@
+# Chouchou
+
+![Chouchou](https://i.imgur.com/BYk2n5C.jpeg)
```

Comment:
```suggestion
![Chouchou](https://i.imgur.com/BYk2n5Ch.jpeg)
```

This uses a lower resolution version of the image that fits better into github's markdown rendering.